tools/windows-node-installer/pkg/cloudprovider/factory.go

```diff
@@ -41,15 +41,15 @@ func CloudProviderFactory(kubeconfigPath, credentialPath, credentialAccountID, r
 	// File, dir, credential account sanity checks.
 	kubeconfigPath, err := makeValidAbsPath(kubeconfigPath)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("invalid path for kubeconfig file, %v", err)
```

Comment:
This is not always true right? You could've gotten error saying the path doesn't exist or something like that? The message is misleading in those scenarios

Reply:
I did hesitate about the message here. It could be an empty path, does not exist, or other errors. All of which I thought could be categorized as an invalid path.